### PR TITLE
fix: updated to use correct wif configuration

### PR DESCRIPTION
**Pull request by bradegler**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -45,8 +43,8 @@ jobs:
         name: 'Authenticate to Google Cloud'
         uses: 'google-github-actions/auth@c4799db9111fba4461e9f9da8732e5057b394f72' # ratchet:google-github-actions/auth@v0
         with:
-          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
-          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+          workload_identity_provider: '${{ vars.WIF_PROVIDER }}'
```
Comment (verbanicm) [inline .github/workflows/release.yml:46]: Should we make this the production environment also?

Comment (bradegler) [reply .github/workflows/release.yml:46]: oh, I missed those, I'll update them. I'm eventually going to rip prod deployment out of this workflow and into infra.

Comment (bradegler) [reply .github/workflows/release.yml:46]: oh, I missed those, I'll update them. I'm eventually going to rip prod deployment out of this workflow and into infra.

Comment (verbanicm) [reply .github/workflows/release.yml:46]: can you do it now so we dont break existing production?

Comment (verbanicm) [reply .github/workflows/release.yml:46]: also, theres probably some IAM permissions in infra needed here.

Comment (bradegler) [reply .github/workflows/release.yml:46]: great, more PRs! :)

**Review by verbanicm — APPROVED** [review]

**Pull request merged**